### DISCUSSION TITLE
Clean up GitHub Actions workflows

### DISCUSSION THREAD

**Discussion by wigging** [ORNL/Zambeze#145] [General]
Some of the workflows don't seem necessary for this project.

There is the "Command Mode Formatting" in `apply-formatting.yml` that apparently formats Python and C++ code. I don't see any C++ in this project and the Python formatting is handled by black. Can someone explain why this workflow is needed and can it be removed from the GitHub Actions for this project?

There is the "Pylint" workflow in `pylint-check.yml` which seems redundant since flake8 is being used for linting in this project. What is pylint providing that flake8 is not providing? Can this workflow be removed from this project too?

**Comment by wigging** [comment]
@tskluzac Let's figure out which workflows can be completely deleted. From the meeting earlier today and based on my original post  it sounds like the following can be removed:
- apply-formatting.yml
- pyre-check.yml
- pylint-check.yml

**Comment by tskluzac** [comment]
Yeah, I think format-check.yml covers the linting pieces, so we should be able to remove the files you've mentioned above.

**Reply by wigging** [reply]
Ok, I'll draft a pull request to start cleaning up the tests. We can use the pull request to discuss further changes.
